src/Documentation/Orleans-Configuration-Guide/Client-Configuration.md

```diff
@@ -134,17 +134,42 @@ config.RegisterStreamProvider<SimpleMessageStreamProvider>("SimpleMessagingStrea
         { "PubSubType", "ExplicitGrainBasedAndImplicit" }
     });
 
-// Initialize client using config defined above
-while (!GrainClient.IsInitialized)
+IClusterClient client = null;
+while (true)
 {
     try
     {
-        GrainClient.Initialize(config);
+        // Build a client and then connect it to the cluster.
+        client = new ClientBuilder()
+            .UseConfiguration(config)
+            .ConfigureServices(
+                services =>
+                {
+                    // Services can be provided to the client here. These services are made
+                    // available via dependency injection.
+                    // ConfigureServices can be called multiple times for a single
+                    // ClientBuilder instance.
```

Comment:
I was just thinking adding an example about how user can inject their own service into DI using ClientBuilder. Literally just 
`services.AddSingleton<YourCustomService>()`, 
But maybe I was overthinking. And current example is clear enough. I will leave it to you to decide what you want to do.  I will merge it at the end of the day or tomorrow regardless.